network/http/variables.tf

```
@@ -90,6 +96,12 @@ variable "http_regex_matched_transformation_function" {
   default     = ".max(over='5m')"
 }
 
+variable "http_regex_matched_lasting_duration" {
+  description = "Duration that indicates how long we wait before triggering the http_regex_matched alert. Specified as s, m, h, d"
+  type        = string
+  default     = "1s"
+}
+
```

please follow current naming: https://github.com/claranet/terraform-signalfx-detectors/blob/master/cloud/aws/alb/variables.tf#L123

```suggestion
variable "http_regex_matched_lasting_duration_seconds" {
  description = "Minimum duration that conditions must be true before raising alert (in seconds)"
  type        = number
  default     = 60
}

```

I don't understand why you choose `1s` which will not make this detector more tolerant (which is what you want to achieve I think) so I suggested `60s` but feel free to change as you prefer.